ovos_bus_client/util/scheduler.py

```diff
@@ -338,14 +338,21 @@ def shutdown(self):
             self.bus.remove_all_listeners('mycroft.scheduler.remove_event')
             self.bus.remove_all_listeners('mycroft.scheduler.update_event')
             # Wait for thread to finish
-            self.join()
+            self.join(30)
```

Comment:
I am mildly afraid of this magic number but I don't have a better idea

Reply:
Or, to be clearer, I don't think that it would help to name the variable more than a tiny bit, because identifying this as the hung thing would be the hard part

Reply:
arbitrary number that is lots of time but likely less than something like a systemd timeout.. Theoretically we should only have to wait 0.5s + the time it takes to run `check_state`, but in highly constrained cases like an overloaded Pi, I've seen things slow down a lot